trace/backend_collector_test.go

```diff
@@ -89,7 +90,7 @@ func TestObfuscate(t *testing.T) {
 		},
 	}
 
-	col := NewBackendCollector(fakeSvc, fakeLrn, mockClient, kgxapi.Inbound, nil)
+	col := NewBackendCollector(fakeSvc, fakeLrn, mockClient, kgxapi.Inbound, nil, nil)
```

Comment:
How did this compile before?

Reply:
It didn't!  We need to enable CircleCI test runs on the akita-cli repository to catch things like this.